frontend/src/metabase/dashboard/utils.js

```diff
@@ -72,8 +72,10 @@ export function showVirtualDashCardInfoText(dashcard, isMobile) {
   }
 }
 
-export function getNativeDashCardEmptyMappingText(parameter) {
-  if (isDateParameter(parameter)) {
+export function getNativeDashCardEmptyMappingText(parameter, type) {
+  if (type === "action") {
+    return t`Add a variable to this action to connect it to a dashboard filter.`;
```

Comment:
Maybe it should say "page filter" instead of "dashboard filter"?

Reply:
Also, feels like we should better add a dedicated function for actions instead of extending this one or give `getNativeDashCardEmptyMappingText` a more generic name

First of all, doesn't seem like any `type === "question"` code is relevant to `type === "action"` cases and vice-versa. Second, the "native dash card" part can be confusing for actions. We kinda have them for query actions under the hood, but that'd become wrong once HTTP actions are in

Reply:
I think it makes sense for actions, but I agree the function name should be more generic, I've changed it to `getDashCardEmptyMappingText`